db-client/src/db.ts

```diff
@@ -81,19 +83,40 @@ export class DB {
     return this.client.getWithVersion(this.ctx, key);
   }
 
-  public async startPolling<T extends Serializable = any>(_key: string): Promise<Versioned<T | undefined>> {
-    throw new Error('Unimplemented');
+  /**
+   * Polls on updates to specified keys since specified versions.
+   * @see KeyedVersions
+   * @see KeyedPatches
+   */
+  public async poll(
+    keysToVersions: Array<[string, Version]>,
+    opts: PollOptions = {},
+  ): Promise<Array<[string, Patch[]]>> {
+    return this.client.poll(this.ctx, keysToVersions, opts);
```

Comment:
Nit: `async` _and_ not `await`?

Reply:
I agree with @vogre that this way is cleaner.